frontend/public/locales/en/access-to-governmental-benefits.json

```diff
@@ -0,0 +1,71 @@
+
+{
+  "access-to-governmental-benefits": {
+    "confirm": {
+      "page-title": "Application submitted",
+      "alert-heading": "You have successfully applied for the Canadian Dental Care Plan!",
+      "your-info-has-been-updated": "Your information has been updated.",
+      "button": {
+        "back": "Back"
+      }
+    },
+    "view": {
+      "page-title": "Access to governmental benefits",
+      "intro-text": "You have indicated that you have access to the following governmental benefits.",
+      "provincial-or-territorial-dental-benefit": "Provincial or Territorial dental benefit:",
+      "alberta-child-health-benefits": "Alberta Child Health Benefits",
+      "federal-benefits-dental-benefits": "Federal benefits dental benefits:",
+      "dental-service-program": "Dental  Service Program",
```

Comment:
"Alberta Child Health Benefits" and "Dental Services Program" (and their French names) should be coming from the lookup service (`getAllProvincialTerritorialSocialPrograms` and `getAllFederalSocialPrograms`) - no need to define i18n keys for them.

Reply:
Removed: 3e3028732d9cee2c8c61af92f40a0aa376b10515.